### PR TITLE
Instrument error logs

### PR DESCRIPTION
**Pull request by mmatczuk**

### CODE REVIEW

File: cmd/forwarder/run/run.go [hunk 1]
```diff
@@ -256,6 +260,22 @@ func (c *command) runE(cmd *cobra.Command, _ []string) (cmdErr error) {
...
+		Namespace: c.httpProxyConfig.PromNamespace,
+		Name:      "errors_total",
+		Help:      "Number of errors",
+	}, []string{"name"})
+
+	if err := c.promReg.Register(m); err != nil {
+		return nil, err
+	}
+
+	return func(name, format string) {
```
Comment (Choraden) [inline cmd/forwarder/run/run.go:274]: nit: `format` is not used -> `_`

Comment (Choraden) [reply cmd/forwarder/run/run.go:274]: Don't we have a linter for that? 🤔

**Review by Choraden — APPROVED** [review]
LGTM

nit: I would squash `log/stdlog: add OnError()` with `log/stdlog: add name and format to onError`, I don't see a value of keeping them split.

**Comment by mmatczuk** [comment]
Yeah I should rewrite it.

File: cmd/forwarder/run/run.go (hunk 1 above)
Comment (mmatczuk) [reply cmd/forwarder/run/run.go:274]: It's for the future, probably not worth it.

**Comment by mmatczuk** [comment]
Sent patchset v2

**Pull request merged**